tests/containers/buildah.pm

```diff
@@ -41,7 +41,7 @@ sub run_tests {
     # network failure
     if ($runtime eq 'podman') {
         record_info('Test', "Install random package in the container");
-        assert_script_run("buildah run $container -- zypper in -y cowsay", timeout => 300);
+        assert_script_run("buildah run $container -- zypper in -y cowsay", timeout => 600);
```

Comment:
```suggestion
        assert_script_run("buildah run $container -- zypper in -y perl", timeout => 600);
```

Since this test expects a simple package and cowsay pulls 5 new packages, we can simplify further by just installing perl.

```
The following 5 NEW packages are going to be installed:
  cowsay libgdbm6 libgdbm_compat4 perl perl-base
```
https://openqa.suse.de/tests/18598753/file/serial_terminal.txt

Reply:
> I see. How about `update-test-trivial` "Package for testing the update stack during product development"?

This may be a problem when I add a reproducer to the container tests.  Is this package available on Leap & Tumbleweed?

Reply:
Yes, it's available on Leap, Tumbleweed, Slowroll and also SLE and SLFO, see https://build.suse.de/search?search_text=update-test-trivial

Reply:
> Yes, it's available on Leap, Tumbleweed, Slowroll and also SLE and SLFO, see https://build.suse.de/search?search_text=update-test-trivial

Ok, let's use it.

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/22822